frame/support/src/macros.rs

```diff
@@ -24,38 +24,38 @@ macro_rules! impl_account_data {
 			$($gtypebound)*
 		)?
 		{
-			pub free: Balance,
-			pub reserved: Balance,
+			pub free_ring: Balance,
```

Comment:
Prefer to keep as it was.

Old name is ok, there could be migration works and apps compatible issues